build.gradle

```diff
@@ -29,7 +28,20 @@ targetCompatibility = 1.8
 repositories {
     mavenCentral()
     maven {
-        url "http://artifactory.terasology.org/artifactory/virtual-repo-live"
+        def repoViaEnv = System.getenv()["RESOLUTION_REPO"]
```

Comment:
Same here, should we use something like `TERASOLOGY_MVN_RESOLUTION_REPO` to be sure?